doc/release-notes-31278.md

```diff
@@ -0,0 +1,7 @@
+Updated RPCs
+---
+- The `settxfee` RPC is now **deprecated** and will be removed in Bitcoin Core 31.0. This RPC allowed users to set a static fee rate for wallet transactions, which is discouraged in a dynamic fee environment as it can lead to overpaying or underpaying. Users should instead rely on fee estimation or specify a fee rate per transaction using the `fee_rate` argument in RPCs like `fundrawtransaction`, `sendtoaddress`, `send`, `sendall`, and `send_many`. (#31278)
+
+Updated settings
+------
+- The `-paytxfee` setting is now **deprecated** and will be removed in Bitcoin Core 31.0. This option set a static fee rate for all wallet transactions when starting the Bitcoin node, which is discouraged in a dynamic fee environment as it can lead to overpaying or underpaying. Users should instead rely on fee estimation or specify a fee rate per transaction using the `fee_rate` argument in RPCs like `fundrawtransaction`, `sendtoaddress`, `send`, `sendall`, and `send_many`. (#31278)
```

Comment:
Same:
```suggestion
- The `-paytxfee` setting is now **deprecated** and will be removed in Bitcoin Core 31.0. This option set a static fee rate for all wallet transactions when starting the Bitcoin node, which is discouraged in a dynamic fee environment as it can lead to overpaying or underpaying. Users should instead rely on fee estimation or specify a fee rate per transaction using the `fee_rate` argument in RPCs like `fundrawtransaction`, `sendtoaddress`, `send`, `sendall`, and `sendmany`. (#31278)
```